### ISSUE TITLE
Older Dash Buttons

### ISSUE THREAD

**Issue by audiether** [danimtb/dasshio#99]
I have a bunch of really old Dash buttons (Rev 01).  Some of them still have working batteries, apparently (white light comes on before blue LED setup light.

Firmware: v0.9.168

I followed the process to connect to the AP and used the URL http://192.168.0.1/?amzn_ssid=<wifi_network_name>&amzn_pw=<wifi_network_pw> (replacing correct values).

One some of the buttons a red light comes on.  On others the light just goes off.

However, when I try to press the button, the white light flashes then a red light comes on.

I have blocked Internet access at the router for the MAC address.

Does anyone have any ideas on getting these to work?

**Comment by vekexasia** [comment]
same issue

**Comment by anatwick** [comment]
did you ever figure this out by chance?  I'm having an issue with mine as well

**Comment by audiether** [comment]
> did you ever figure this out by chance? I'm having an issue with mine as well

Sorry, I gave up.  Turns out all of my devices had dead batteries and the batteries were soldered on the unit.  Tried to replace one, then gave up.

I replaced all my automations with the Shortcuts App on my iPhone.  Instead of pressing buttons to get a script to do something, I now tell Siri to do it.  Actually like it even better because I do not need the button anywhere near me.

Good luck!
